core/src/main/java/com/linecorp/armeria/server/cors/CorsService.java

```diff
@@ -59,6 +59,19 @@
     private static final String DELIMITER = ",";
     private static final Joiner HEADER_JOINER = Joiner.on(DELIMITER);
 
+    /**
+     * Check for a CORS preflight request.
+     *
+     * @param request the HTTP request with CORS info
+     *
+     * @return {@code true} if HTTP request is a CORS preflight request
+     */
+    public static boolean isCorsPreflightRequest(final HttpRequest request) {
```

Comment:
How about `isPreflightRequest()`? (because the class name already includes 'CORS')